docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+  mongo:
+    ports:
+      - "27017:27017"
+    container_name: plasmidi-mongo
+    restart: always
+    logging:
+      options:
+        max-size: 1g
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoadmin
+      MONGO_INITDB_ROOT_PASSWORD: secret
+      MONGO_INITDB_DATABASE: plasmidi
+    image: mongo
```

Comment:
local mongo db server for dev. 

I made this because I don't want a mongo database on my computer. 

https://hub.docker.com/_/mongo

this mentions how to seed the db when it's created, so it's probably pretty easy to create seed data